Drop-In/src/main/java/com/braintreepayments/api/DropInActivity.java

```diff
@@ -133,27 +143,80 @@ void onUIEvent(DropInUIEvent event) {
                 // to see if this parameter is necessary
                 updateVaultedPaymentMethodNonces(false);
                 break;
+            case DropInUIEventType.DISMISS_VAULT_MANAGER:
+                showSelectPaymentMethodFragment();
+                break;
         }
     }
 
+    void onDeleteVaultedPaymentMethodSelected(DeleteVaultedPaymentMethodNonceEvent event) {
+        final PaymentMethodNonce paymentMethodNonceToDelete = event.getPaymentMethodNonceToDelete();
+
+        PaymentMethodItemView dialogView = new PaymentMethodItemView(this);
+        dialogView.setPaymentMethod(paymentMethodNonceToDelete, false);
+
+        new AlertDialog.Builder(this,
+                R.style.Theme_AppCompat_Light_Dialog_Alert)
+                .setTitle(R.string.bt_delete_confirmation_title)
+                .setMessage(R.string.bt_delete_confirmation_description)
+                .setView(dialogView)
+                .setPositiveButton(R.string.bt_delete, new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        getDropInClient().sendAnalyticsEvent("manager.delete.confirmation.positive");
+                        removePaymentMethodNonce(paymentMethodNonceToDelete);
+                    }
+                })
+                .setOnDismissListener(new DialogInterface.OnDismissListener() {
+                    @Override
+                    public void onDismiss(DialogInterface dialog) {
+                        getDropInClient().sendAnalyticsEvent("manager.delete.confirmation.negative");
+                    }
+                })
+                .setNegativeButton(R.string.bt_cancel, new DialogInterface.OnClickListener() {
```

Comment:
I think you can pass `null` for the dialog click listener. It should function the same as setting an empty listener.

Reply:
Good call - changed this to null